data/tlg0007/tlg071/tlg0007.tlg071.perseus-eng3.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
    <teiHeader xml:lang="eng">
       <fileDesc>
          <titleStmt>
-            <title rend="italic" xml:lang="lat">Quomodo quis suos in virtute sentiat profectus</title>
+            <title xml:lang="lat">Quomodo quis suos in virtute sentiat profectus</title>
```

Comment:
What's the book say on this? I have How a Man May Become Aware of his Progress in Virtue. Always defer to what's on the page for the edition/translation in question. The cts work file will resolve the canonical naming.

Reply:
Which is to say also check your cts work file for the correct names/labels.

Reply:
Checked work CTS file- it's in English there and also reflects what's in the edition so I'll start making that change (the title is in Latin in the English files)